### DISCUSSION TITLE
Where to post errors/fixes?

### DISCUSSION THREAD

**Discussion by iBeleave** [geodesicsolutions-community/geocore-community#219] [Q&A] [answered]
I am updating my site to this Community Edition so that I can move to PHP 8.  As I run into issues (and fix them), wondering if I should post the issues here?  I don't know how to use git so I am not able to contribute that way. 

Our site uses some features that are lesson common, so it's likely I will run into things.  For instance, already fixed one with the account balance addon.  Anyway, just want to share what I find, in case it's useful to anyone.  Thoughts on the place/method of doing that?

**Comment by vicos59** [comment, marked as answer]
Hi,

You can Post issues here:

https://github.com/geodesicsolutions-community/geocore-community/issues

General questions and conversations can be here in the Discussions section.

**Comment by iBeleave** [comment]
perfect - thanks!
